docs/02-module-block-2/02-learning-goals.adoc

```diff
@@ -2,19 +2,77 @@
+
+Den Teilnehmer:innen ist bewußt, dass Referenzarchitekturen begleitend zu diesen Phasen auch die folgenden Aufgabenbereiche mit geeigneten Komponenten abdecken müssen:
+
+- Data Discovery - Information bereitstellen, welche Daten es gibt, wie sie aufbereitet und wo sie zu finden sind
+- Data Governance - Erarbeiten und Überwachen allgemein gültiger Regeln für die Verarbeitung der Daten
+- Data Security - Gewährleisten eines angemessenen Grads an Datenschutz
+- Data Quality - Gewährleisten eines angemessenen Grads an Datenqualität
+
+[[LZ-2-3]]
+==== LZ 2-3 - Referenzarchitekturen zur Anwendung von Verfahren der KI und des ML
+Die Teilnehmer:innen können die folgenden Phasen der Datenverarbeitung zur Anwendung von Verfahren der KI und des ML unterscheiden und wissen, welche Komponenten in den jeweiligen Phasen typischerweise zum Einsatz kommen:
+
+- Data Transformation - Konvertieren der Rohdaten in ein geeignetes Format für das Model Training (Data Labeling, Data Science Libraries, ...)
+- Model Training und Development - Iteratives Trainieren der Modelle mit den Daten, Optimieren und Automatisieren der Abläufe für das Trainieren der Modelle (Feature Store, Model Registry, ML/DL Framework, Workflow Manager, ...)
+- Model Inference - Anwenden der Modelle auf die zu analysierenden Daten. Monitoring der Verarbeitung
+- Integration - Integration der Analyse-Verfahren und Ergebnisse in Anwendungen
+
+[[LZ-2-4]]
+==== LZ 2-4 - Architekturentscheidungen anhand von Referenzarchitekturen
+Die Teilnehmer:innen können Architekturentscheidungen speziell zu den folgenden Fragestellungen anhand von Referenzarchitekturen diskutieren:
 
+- ob ein zentrales/monolithisches oder dezentrales/förderiertes System zu wählen ist {Single Point of Truth, Canonical Data Model, Master Data Management}
+- ob ein Deployment in der Cloud (ggfs. Cloud-Ready), hybrid oder on-Premise vorzuziehen ist
+- ob ein einheitliches Schema für die auszuwertenden Daten vorgegeben werden soll
+- ob eine eigenständige Entwicklung von Komponenten erforderlich ist oder das Kaufen und Konfigurieren von Standardkomponenten ausreicht
+- ob die Integration von Komponenten selbst vorgenommen oder auf die Verwendung bereits integrierter Komponenten vertraut wird
+- ob Stream-, Batch-Processing (ggfs Micro-Batch) oder beides unterstützt werden muss
+- welches Bezahlmodell (Pay as you go, subscription fee, buy) Vorteile bietet
+- welche Form der Skalierbarkeit konkret benötigt wird: Datenvolumen, Geschwindigkeit der Verarbeitung und des Datenzugriffs, Anzahl und Variabilität der Datenquellen, Anzahl der Nutzer, Anzahl und Variabilität der analytischen Auswertungen
+- inwieweit Daten repliziert/redundant gespeichert werden müssen
+- ob eine fachliche Modularisierung (im Sinne von DDD) erforderlich ist
+- welche Rollen und Verantwortlichkeiten vorgesehen werden müssen
+- wie groß Komponenten dimensioniert werden müssen
 // end::DE[]
```

Comment:
Reis benennt in seinem Buch noch explizit Besonderheiten für IoT Systeme. Denke zwar nicht, dass das hier explizit angesprochen werden muss. Ich will es aber der Vollständigkeit halbe anmerken.